backend/templates/index.html

```diff
@@ -0,0 +1,13 @@
+{% load render_bundle from webpack_loader %}
+<!DOCTYPE html>
+<html>
```

Comment:
pretty sure you need `<html lang=en>` and a couple specialized `<meta>` tags in order to ensure that Bootstrap works properly. Check out the template [here](http://getbootstrap.com/getting-started/#template).